src/main/java/kmql/table/ReplicasTable.java

```diff
@@ -30,25 +30,28 @@ public void prepare(Connection connection, AdminClient adminClient) throws Excep
                          + "is_leader BOOLEAN NOT NULL,"
                          + "is_preferred_leader BOOLEAN NOT NULL,"
                          + "is_in_sync BOOLEAN NOT NULL,"
+                         + "seq INT NOT NULL,"
```

Comment:
Can we name it bit more descriptive? How about `replica_order` or `replica_index`?